src/widgets/map/styles/MapMarker.css

```
@@ -0,0 +1,20 @@
+.map__marker {
+  width: 2.292vmax;
+  aspect-ratio: 11/14;
+}
+
+.map__marker__shadow {
+  fill: rgba(0, 0, 0, 0.22);
+}
+
+.map__marker__body {
+  fill: url(#pinGrad);
```

Stylelint (17.3.0)

[error] Expected quotes around "url" function argument (function-url-quotes)

(function-url-quotes)